### PR TITLE
Expose scrape interval.

### PR DESCRIPTION
**Pull request by mrsiano**
Expose the scrape config interval, in order to control it.

motiviation:
by default this value is 1m, when working with grafana and k8s templates,
we cannot see graphs correctly, and they needs to be tuned.

see

### CODE REVIEW

File: examples/prometheus/prometheus.yaml [hunk 1]
```diff
@@ -22,6 +22,9 @@ parameters:
   name: SESSION_SECRET
   generate: expression
   from: "[a-zA-Z0-9]{43}"
+- description: Time interval for scrape config (1m, 5s)
+  name: SCRAPE_INTERVAL
+  value: 15s
```
Comment (jeremyeder) [inline examples/prometheus/prometheus.yaml]: Let's not change the existing default of 1m.  This new variable allows us to override 1m (which we may do in all cases, but still, we do not want to change the default of 1m).

**Comment by smarterclayton** [comment]
I’m not sure why I would want to take this change.  Why is 1m not good enough for 90% of users?

If you want to change it afterwards, you can do so

**Comment by jeremyeder** [comment]
@smarterclayton this gives us an easy way to change the interval when we instantiate a pod from the template (via argument to new-app), and is a noop to existing users of the prometheus template.

**Comment by smarterclayton** [comment]
That's not what we're really optimizing for - we're trying to actively reduce parameterization.  If the number of users expected to change this is less than 50%, then it's not worth adding if it's easy to customize when you need it.  The purpose of the sample is to be the canonical default, not infinitely flexible.

**Comment by mrsiano** [comment]
closing due to required changes. will submit another PR.

**Pull request closed without merging**